### ISSUE TITLE
Match Lando MariaDB version with BRD RDS version

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#4345]
Breaking this out into its own story since it is dependent on an upstream PR:

> Lando MariaDB is using version 10.5 (blocked on upstream PR to Lando) - https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4304

_Originally posted by @ElijahLynn in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4240#issuecomment-780241145_

**Comment by ElijahLynn** [comment]
Upstream PR for Lando version bump is merged now. But need to wait for a new Lando tag to be released now and then we can merge https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4304 but all engineers will need to update Lando.

**Comment by ElijahLynn** [comment]
> Upstream PR for Lando version bump is merged now. But need to wait for a new Lando tag to be released now and then we can merge #4304 but all engineers will need to update Lando.

New tag is out > https://github.com/lando/lando/releases/tag/v3.0.26

We can merge this but I think we might want to do it Friday after demo because engineers will need to update Lando to new tag and `lando rebuild`. And don't want to interrupt their flow until after demo.
